.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -48,6 +48,10 @@ body:
       render: shell
     validations:
       required: true      
+  - type: markdown
```

Comment:
may be put this to the top of the issue template? (e.g. right below the note about security issues) it would be a little frustrating if you already added some data (version, environment description) and than found the issue in the troubleshooting page